test/integration/test_cli_runners.py

```diff
@@ -42,6 +45,8 @@ def start_ssh_docker(container_name, jobs_directory, port=10022, image='agaveapi
                           'nofile=2048:2048',
                           image]
     subprocess.check_call(START_SLURM_DOCKER)
+    if 'openpbs' in image:
+        time.sleep(PBS_STARTUP_DELAY)
     if sys.platform != 'darwin':
         subprocess.check_call(['docker', 'exec', container_name, 'usermod', '-u', str(os.getuid()), 'testuser'])
```

Comment:
```suggestion
        # /home/testuser/.ssh/authorized_keys in the container needs to be owned
        # by testuser, so change the testuser's uid to match. Alternatively, we
        # could `docker cp` the public key file instead of using a bind mount,
        # and then chown it inside the container.
        subprocess.check_call(['docker', 'exec', container_name, 'usermod', '-u', str(os.getuid()), 'testuser'])
```

Reply:
But that's only half the story, you also don't want the bind-mounted outputs with 1001 on the test machine, right ?
```suggestion
        # Change testuser's uid to match current user id. This ensures that /home/testuser/.ssh/authorized_keys
        # is owned by the right user and that job outputs can be cleaned up.
        subprocess.check_call(['docker', 'exec', container_name, 'usermod', '-u', str(os.getuid()), 'testuser'])
```

Reply:
You can still remove files owned by another user if you own the directory where the files are, but you cannot remove their non-empty subdirectorie, so you are right, the `jobs_directory` bind mount could be a problem if the uid don't match.